### PR TITLE
Use new domexception module

### LINKED ISSUE #97: delete() and "?"
```js
const nativeURL = require("url").URL;
const shimmedURL = require("whatwg-url").URL;

const native = new nativeURL("http://host/?param");
native.searchParams.delete("param");
native.href;
//-> http://host/

const shimmed = new shimmedURL("http://host/?param");
shimmed.searchParams.delete("param");
shimmed.href;
//-> http://host/?
```

Pretty sure that the native one is correct. @TimothyGu ?

### PR DESCRIPTION
**Pull request by TimothyGu**
Will be helpful in #100.

Fixes #97.

### CODE REVIEW

**Comment by Sebmaster** [comment]
Seems like this is breaking a test?

**Comment by TimothyGu** [comment]
Ah, right, because the tests weren't updated for the new DOMException definition in IDL. Let's see if this fixes it. (I can't check locally right now.)

**Comment by TimothyGu** [comment]
Okay tests pass now.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ whatwg-url is a full implementation of the WHATWG [URL Standard](https://url.spe
 
 ## Current status
 
-whatwg-url is currently up to date with the URL spec up to commit [0f5395](https://github.com/whatwg/url/commit/0f53958338bbaec3882f902897873da59ba7e8bd), except that we haven't implemented changes relating to the new UTS #46 revision ([dc9d831](dc9d83106cada9af507bf37dee3973de97b020fd), [b128ba9](https://github.com/whatwg/url/commit/b128ba9111c68ad767b472d77d0ada9ef85366ef)) yet. Work on fixing that gap is in progress in [Sebmaster/tr46.js#11](https://github.com/Sebmaster/tr46.js/pull/11).
```
Comment (Sebmaster) [inline README.md]: Should update the w3c tests in this commit as well if we're modifying the readme here.

Comment (TimothyGu) [inline README.md]: The tests won't pass in this commit though (we need both commits).

Comment (Sebmaster) [inline README.md]: Hrm, what if we split the update of the tests into its own commit?

Comment (TimothyGu) [inline README.md]: Sure.

**Pull request merged**